trunner/harness/base.py

```diff
@@ -109,6 +109,9 @@ def __call__(self, flash=False, hard=False):
 
         self._set_flash_mode(flash)
 
+        # Ensuring that first up target will change state before restart
+        time.sleep(0.5)
```

Comment:
On other targets are no impact about that, because construction and way how they change states are different from zynq 7000. On other targets we can easily first up reboot and in this time change state. Zynq7000 need to have to change state as first because it prepares target for what is needed after reboot.

This is `Reset cycle` of zynq7000-zedboard:
![image](https://github.com/phoenix-rtos/phoenix-rtos-tests/assets/49757239/cc1d6087-560a-487d-bf59-bf9f4c4c2491)

And there is reason why `Reboot` is needed and what types of `Signals` can be needed in this situation:
![image](https://github.com/phoenix-rtos/phoenix-rtos-tests/assets/49757239/b6847e43-4251-454c-870c-6adea70c2b1b)

Reply:
Tested for a while 0.05s - I've run about 100 restarts and it seemed to work stable - to verify whether it's not possible to make it a bit shorter.